zones.txt

```diff
@@ -4827,6 +4827,209 @@ ac.zw
 co.zw
 com.zw
 org.zw
+001.рус
```

Comment:
Who owns these numeric `рус` domains? What are their purpose?

Reply:
They owned by [webnames](https://www.webnames.ru/en/index.pl) registrar, purpose is unknown because I am not related to this registrar, perhaps for russian regions which some how related to these numbers.